client/wfnews-war/src/main/angular/package.json

```diff
@@ -42,37 +42,37 @@
     "@angular/platform-browser-dynamic": "15.2.9",
     "@angular/router": "15.2.9",
     "@angular/service-worker": "15.2.9",
-    "@angular/youtube-player": "^15.2.9",
+    "@angular/youtube-player": "15.2.9",
     "@busacca/ng-pick-datetime": "10.0.0",
-    "@capacitor-community/fcm": "^5.0.2",
-    "@capacitor/android": "^5.5.1",
-    "@capacitor/app": "^5.0.6",
-    "@capacitor/app-launcher": "^5.0.2",
-    "@capacitor/browser": "^5.1.0",
-    "@capacitor/camera": "^5.0.7",
-    "@capacitor/core": "^5.5.1",
-    "@capacitor/device": "^5.0.6",
-    "@capacitor/dialog": "^5.0.6",
-    "@capacitor/filesystem": "^5.1.4",
-    "@capacitor/geolocation": "^5.0.6",
-    "@capacitor/haptics": "^5.0.6",
-    "@capacitor/ios": "^5.5.1",
-    "@capacitor/keyboard": "^5.0.6",
-    "@capacitor/motion": "^5.0.6",
-    "@capacitor/network": "^5.0.6",
-    "@capacitor/preferences": "^5.0.6",
-    "@capacitor/push-notifications": "^5.1.0",
-    "@capacitor/splash-screen": "^5.0.6",
-    "@capacitor/status-bar": "^5.0.6",
+    "@capacitor-community/fcm": "5.0.2",
+    "@capacitor/android": "5.6.0",
+    "@capacitor/app": "5.0.6",
+    "@capacitor/app-launcher": "5.0.6",
+    "@capacitor/browser": "5.1.0",
+    "@capacitor/camera": "5.0.8",
+    "@capacitor/core": "5.6.0",
+    "@capacitor/device": "5.0.6",
+    "@capacitor/dialog": "5.0.6",
+    "@capacitor/filesystem": "5.2.0",
+    "@capacitor/geolocation": "5.0.6",
+    "@capacitor/haptics": "5.0.6",
+    "@capacitor/ios": "5.6.0",
+    "@capacitor/keyboard": "5.0.6",
+    "@capacitor/motion": "5.0.6",
+    "@capacitor/network": "5.0.6",
+    "@capacitor/preferences": "5.0.6",
+    "@capacitor/push-notifications": "5.1.0",
+    "@capacitor/splash-screen": "5.0.6",
+    "@capacitor/status-bar": "5.0.6",
     "@capawesome/capacitor-background-task": "^5.0.0",
     "@ckeditor/ckeditor5-angular": "4.0.0",
     "@ckeditor/ckeditor5-build-decoupled-document": "35.0.1",
-    "@ionic-native/core": "^5.36.0",
-    "@ionic-native/http": "^5.36.0",
-    "@ionic/angular": "^7.4.0",
-    "@ionic/core": "^7.4.0",
-    "@ionic/pwa-elements": "^3.2.2",
-    "@ionic/storage-angular": "^4.0.0",
+    "@ionic-native/core": "5.36.0",
```

Comment:
we dont need ionic-core & ionic-http anymore. so these can be removed I believe.